com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/HTML5Bundler.java

```diff
@@ -426,15 +426,50 @@ public void bundleApplication(Project project, Platform platform, File bundleDir
         if (variant.equals(Bob.VARIANT_RELEASE)) {
             symbolsName = "dmengine_release.js.symbols";
         }
-        String zipDir = FilenameUtils.concat(project.getBinaryOutputDirectory(), Platform.JsWeb.getExtenderPair());
-        File bundleSymbols = new File(zipDir, symbolsName);
-        if (!bundleSymbols.exists()) {
-            zipDir = FilenameUtils.concat(project.getBinaryOutputDirectory(), Platform.WasmWeb.getExtenderPair());
-            bundleSymbols = new File(zipDir, symbolsName);
-        }
-        if (bundleSymbols.exists()) {
-            File symbolsOut = new File(appDir.getParentFile(), enginePrefix + ".symbols");
-            FileUtils.copyFile(bundleSymbols, symbolsOut);
+        // Copy symbols for relevant web pairs (js-web, wasm-web, wasm_pthread-web)
+        String[] webSymbolPairs = new String[] {
+            Platform.JsWeb.getExtenderPair(),
+            Platform.WasmWeb.getExtenderPair(),
+            Platform.WasmPthreadWeb.getExtenderPair()
+        };
+        for (String pair : webSymbolPairs) {
+            String dir = FilenameUtils.concat(project.getBinaryOutputDirectory(), pair);
+            File expected = new File(dir, symbolsName);
+
+            File srcSymbols = null;
+            if (expected.exists()) {
+                srcSymbols = expected;
+            } else {
+                // Try to discover any *.js.symbols in the platform directory
```

Comment:
Remind me again, how are we naming the .js.symbols?

Reply:
Why aren't we always just searching for *.js.symbols? Why have two code paths?

Reply:
fixed